src/app/utils/createElement.ts

```diff
@@ -0,0 +1,39 @@
+import { ElementConfig } from '../interfaces/elementConfig'
```

Comment:
From 'interfaces'

In interfaces folder add index.ts and inside index.ts add export * from customHtmlElement